test/controllers/votes_controller_test.rb

```diff
@@ -0,0 +1,15 @@
+require 'test_helper'
+
+class VotesControllerTest < ActionController::TestCase
+  test "vote on post" do
+    user = FactoryGirl.create(:user)
+    post = FactoryGirl.create(:post, user: user)
+    user2 = FactoryGirl.create(:user)
+    sign_in user2
+    assert_difference 'post.votes.count' do
+      post :create, :user_id => user2.id, :votable_id => post.id, :votable_type => 'Post'
```

Comment:
This is referencing the post variable.

There are some keywords that are reserved and you can't use them, or if you do it will cause problems.  This is one such example.